src/commands/create.js

```diff
@@ -53,7 +53,13 @@ export default async () => {
 
   const isYarn = shouldUseYarn()
 
-  console.log('=> Installing dependencies...')
+  /**
+   * Corbin Matschull (cgmx) - basedjux@gmail.com
+   *
+   * Deps install output will now show WHAT dependency manager-
+   * - you're using before the install steps.
+   */
```

Comment:
Let's refrain from these types of comments in the code unless it's to explain complex logic.

Reply:
Not a problem. Keep comments to complex logic. 👍